packages/venia-concept/src/.storybook/config.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { configure, addDecorator } from '@storybook/react';
+import { Adapter } from '@magento/venia-drivers';
+import store from '../store';
+import '../index.css';
+import { PeregrineContextProvider } from '@magento/peregrine';
+
+function loadStories() {
```

Comment:
why use not an arrow function?

Reply:
I can change it to an arrow function, but I just copied the config from `venia-ui`. But yeah, I can change it